redis/tests/test_basic.rs

```diff
@@ -1100,11 +1139,11 @@ fn test_zrandmember() {
     let results: Vec<String> = con.zrandmember(setname, Some(-5)).unwrap();
     assert_eq!(results.len(), 5);
 
-    let results: Vec<String> = con.zrandmember_withscores(setname, 5).unwrap();
```

Comment:
old Vec<String> test works on resp2 but, doesn't work on resp3.

Reply:
please also keep the RESP2 test.